Home/index.html

```diff
@@ -246,6 +248,39 @@ <h4>
             </div>
         </div>
     </section>
+
+    <section id="contact" class="form_section">
+        <div class="form_text">
+            <h2>
+                I'm always interested in hearing about new projects, so if you'd like
+                to chat please get in touch.
+            </h2>
+        </div>
+        <form action="https://formspree.io/f/mleznykr" method="POST" class="form">
+
+            <input class="form_input form_fullname" type="text" placeholder="Full name" name="user_last_name" required
+                maxlength="60" />
```

Comment:
- [ ] Form is looking great! However, I would suggest having a `maxlength` of 30 characters, as this is an activity requirement:

![Captura de pantalla 2022-05-31 a la(s) 7 31 35 p  m](https://user-images.githubusercontent.com/83717285/171305016-1b6f5fe4-5cba-46f0-917a-932acd01a33d.png)